packages/documentation-site/patternfly-docs/content/PatternFly-AI/patternfly-mcp.md

```diff
@@ -0,0 +1,95 @@
+---
+id: PatternFly MCP
+section: PatternFly-AI
+--- 
+import '../design-guidelines/components/components.css';
+
+This guide provides an overview of the PatternFly MCP server, including its benefits and instructions for setting up the tool.
+
+For full technical documentation, setup instructions, and to contribute, visit our [PatternFly MCP GitHub Repository](https://github.com/patternfly/patternfly-mcp).
+
+## What is the PatternFly MCP?
+The **PatternFly MCP** is a new tool designed to integrate our design guidelines, component documentation, and accessibility best practices directly into your AI-powered development environment. 
```

Comment:
Seeing "new" in our docs immediately makes me think we're building in tech debt as this becomes dated.

```suggestion
The **PatternFly MCP** is a tool designed to integrate our design guidelines, component documentation, and accessibility best practices directly into your AI-powered development environment. 
```